env.file

```diff
@@ -4,4 +4,5 @@ WEB_OIDC_CLIENT_ID=teammail-web
 OIDC_SCOPES=openid,profile,email,offline_access
 APP_GRID_AVAILABLE=supported
 FCM_AVAILABLE=supported
-IOS_FCM=supported
\ No newline at end of file
+IOS_FCM=supported
+FORWARD_WARNING_MESSAGE=
```

Comment:
Are space supported? `,` supported?

Reply:
Yes it has support. Attributes in `env` are only distinguished by line (`\n`)